ansible/roles/kubernetes/defaults/main.yml

```diff
@@ -2,3 +2,5 @@
 kubernetes_kubelet_version: 1.18.6-00
 kubernetes_kubeadm_version: 1.18.6-00
 kubernetes_kubectl_version: 1.18.6-00
+
+os_modules_load_list: /etc/modules
```

Comment:
If `/etc/modules-load.d/br_netfilter.conf` is to be used, this can be removed as that path would be the same on all distributions.